README.md

```diff
@@ -245,6 +246,42 @@ Consider passing these CLI flags on Devnet startup:
 
 Devnet can be restarted by making a `POST /restart` request. All of the deployed contracts, blocks and storage updates will be restarted to the empty state. If you're using [the Hardhat plugin](https://github.com/Shard-Labs/starknet-hardhat-plugin#restart), run `await starknet.devnet.restart()`.
 
+## Advancing time
+
+Block timestamp can be manipulated by seting the exact time or seting the time offset. Timestamps methods won't generate a new block, but they will modify the time of the following blocks. All values should be set in [Unix time](https://en.wikipedia.org/wiki/Unix_time) and seconds.
+
+### Set time
+
+Sets the exact time of the next generated block. All blocks afterwards will keep a set offset.
+
+```
+POST /set_time
+{
+    "time": TIME_IN_SECONDS
+}
+```
+
+Warning: block time can be set in the past and lead to unexpected behaviours!
```

Comment:
behaviours; shouldn't be plural

Reply:
[c763567](https://github.com/Shard-Labs/starknet-devnet/pull/96/commits/c7635675df5998e6dd48cab1d29ec3b90e444707)